### DISCUSSION TITLE
Does Afreeca still work with streamlink?

### DISCUSSION THREAD

**Discussion by derekchen60** [streamlink/streamlink#4577] [Q&A]
Hi all,

Just wondering if Afreeca still works with streamlink. I installed the latest version via Powershell. As an example command of what I us, I simply use "streamlink https://play.afreecatv.com/streamername/241028989". Are there any arguments I need specifically for Afreecatv? I am unable to open any streams and I receive the error "Unable to open URL (('Connection aborted, FileNotFound(2, No such file or directory;))) "

Thanks!

**Comment by bastimeyer** [comment]
Do not report plugin issues on the discussion forum.
Open a new plugin issue thread on the issue tracker and fill in all the required details.
https://github.com/streamlink/streamlink/issues/new/choose
